chef/cookbooks/glance/recipes/api.rb

```diff
@@ -91,13 +91,7 @@
 
 network_settings = GlanceHelper.network_settings(node)
 
-glance_stores = ["glance.store.filesystem.Store",
-                 "glance.store.http.Store",
-                 "glance.store.cinder.Store",
-                 "glance.store.rbd.Store",
-                 "glance.store.swift.Store"]
-
-glance_stores << "glance.store.vmware_datastore.Store" unless node[:glance][:vsphere][:host].empty?
+node[:glance][:glance_stores] << "glance.store.vmware_datastore.Store" unless node[:glance][:vsphere][:host].empty?
```

Comment:
I'd prefer to use a variable that is the concatenation of `node.default[:glance][:glance_stores]` and the vmware store instead of changing `node[:glance][:glance_stores]`, as this could break later on if a store gets removed (as soon as the attribute isn't default anymore).